docs/get-started/v2/get-started/prebuilt/prebuilt.mdx

```diff
@@ -3,13 +3,15 @@ title: 100ms Prebuilt
 nav: 4.3
 ---
 
-100ms Prebuilt is a feature-rich video conferencing/live streaming [web app](https://github.com/100mslive/100ms-web/) made using our prebuilt components. Prebuilt allows users to join sharable room links via desktop, meeting invites, or as an embeddable experience using an [iFrame](/docs/get-started/v2/get-started/prebuilt/prebuilt#share-or-embed-as-iframe), or as a native component in your apps.
+100ms Prebuilt link is a feature-rich video conferencing/live streaming [web app](https://github.com/100mslive/100ms-web/).
```

Comment:
intro is outdated ("web app"). is that fixed in the base branch?

Reply:
naah base branch content is on the left. 

let me comeback with better intro